ray-operator/Makefile

```diff
@@ -63,10 +63,11 @@ fumpt: gofumpt
 	$(GOFUMPT) -l -w ../
 
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+WHAT=$(shell go list ./... | grep -v /test/)
```

Comment:
can WHAT variable be initialised with a `?=`

Reply:
Done, PTAL.